crates/typst-pdf/src/tags/groups.rs

```diff
@@ -620,4 +630,20 @@ impl GroupKind {
             GroupKind::Standard(_, lang) => lang,
         })
     }
+
+    /// Whether this group should be ignored when searching for a semantic
+    /// parent or child group.
+    pub fn is_non_semantic(&self) -> bool {
```

Comment:
I think flipping the name and return value would be clearer, in particular, as all uses use it with `!`.